src/main/resources/db/db.changelog-master.xml

```diff
@@ -639,6 +639,11 @@
     <changeSet author="pawel (generated)" id="1567594875780-11">
         <dropForeignKeyConstraint baseTableName="document_task" constraintName="FKs41maqg7j4qgiewae7qfb6qe5"/>
     </changeSet>
+    <changeSet author="tomelliott (generated)" id="1570622067051-10">
```

Comment:
This should be an additional migration

Reply:
As discussed with @linusnorton - this changeset needs to be moved into correct ordering